clients/admin-ui/src/pages/subject-request/[id].tsx

```diff
@@ -0,0 +1,101 @@
+import {
```

Comment:
Sure, I was unfamiliar with it when I started working on this. It's a feature of Nextjs. Directories within the `pages` directory of a Nextjs app will get turned into a route. So app now has a `subject-request` route (`example.com/subject-request`). Any components within that directory will get turned into pages. The `[id].tsx` syntax is for dynamic routes. It effectively creates a route template at runtime just like fastapi: `/subject-request/{id}`. This will pass the subject request Id to the component so it can pull in the SR data. This documentation goes into further detail: https://nextjs.org/docs/routing/dynamic-routes

Reply:
thanks!